src/Models/LoginSession.php

```diff
@@ -305,4 +323,12 @@ public static function getSessionLifetime(): int
 
         return LoginSession::config()->get('default_session_lifetime');
     }
+
+    /**
+     * @return int
+     */
+    public static function getUpdateThreshold(): int
+    {
+        return LoginSession::config()->get('last_accessed_threshold') ?? 0;
+    }
```

Comment:
Please just remove this method and get the config value directly. If someone sets the configuration value to null they've misconfigured it, and in php `1 + null` is the equivalent of `1 + 0` anyway which is the only context this value is being used in.

Reply:
@GuySartorelli done, i've removed the method